great_expectations/core/usage_statistics/anonymizers/types/base.py

```diff
@@ -15,6 +15,7 @@
 BATCH_REQUEST_OPTIONAL_TOP_LEVEL_KEYS: Set[str] = {
     "data_connector_query",
     "runtime_parameters",
+    "options",
```

Comment:
@Kilo59 This is needed -- otherwise, instantiating an `ExecutionEngine` says that "`options` is not recognized by `__init__()`.